src/recommendationservice/requirements.txt

```diff
@@ -7,23 +7,28 @@
 cachetools==3.1.1         # via google-auth
 certifi==2018.11.29       # via requests
 chardet==3.0.4            # via requests
-google-api-core[grpc]==1.22.0  # via -r requirements.in, google-api-python-client, google-cloud-core, google-cloud-monitoring, google-cloud-trace, opencensus
+google-api-core[grpc]==1.22.0  # via -r requirements.in, google-cloud-core, google-cloud-monitoring, google-cloud-trace, opencensus
 google-api-python-client==1.10.0  # via google-python-cloud-debugger
 google-auth-httplib2==0.0.4  # via google-api-python-client, google-python-cloud-debugger
-google-auth==1.20.1       # via google-api-core, google-api-python-client, google-auth-httplib2, google-python-cloud-debugger
+google-auth==1.20.1       # via google-api-core
 google-cloud-core==0.29.1  # via google-cloud-trace
-google-cloud-monitoring==0.36.0  # via opencensus-ext-stackdriver
-google-cloud-trace==0.24.0  # via opencensus-ext-stackdriver
+google-cloud-monitoring==0.36.0  # via opencensus-ext-stackdriver, opentelemetry-exporter-google-cloud
+google-cloud-trace==0.24.0  # via opencensus-ext-stackdriver, opentelemetry-exporter-google-cloud
 google-python-cloud-debugger==2.15  # via -r requirements.in
 googleapis-common-protos==1.52.0  # via google-api-core
-grpcio-health-checking==1.31.0  # via -r requirements.in
-grpcio==1.31.0            # via -r requirements.in, google-api-core, grpcio-health-checking, opencensus-ext-grpc
```

Comment:
@simonz130 since `requirements.txt` is generated using `pip-compile`, the comments changed and the diff is quite large. I went back to try to make sure that the versions didn't change, but this slightly older version of grpcio was selected because of the `opentelemetry-ext-grpc` dependency.